roles/glance/tasks/main.yml

```diff
@@ -105,3 +105,8 @@
     - logrotate
     - logging
   when: logging.enabled|default('True')|bool
+
+- include: serverspec.yml
+  tags: 
+    - serverspec
+  when: serverspec.enabled|default('False')|bool
```

Comment:
This bit is going to fail, because the serverspec variable is defined in the serverspec role, but the glance role doesn't depend on it.